source/photo_audio_video_file.md

```diff
@@ -0,0 +1,148 @@
+
+![Advanced appearances](images/photo_audio_video_file/advanced_appearances.png)
+
+### Adding advanced appearances in xlsform
+
+You can specify advanced appearances of the "photo" question in XLSForm under
+the appearance column as shown in the following example:
+
+| type  | name       | label                      | appearance |
+| :---- | :--------- | :------------------------- | :--------- |
+| image | sign       | Sign here                  | signature  |
+| image | drawing    | Draw here                  | draw       |
+| image | annotation | Take an image and annotate | annotate   |
+
+## Background audio recording
+
+KoboToolbox now allows you to record audio in the background when you open the
+form. This can be useful in scenarios for example when you are collecting data
+through focus group discussions and you need an audio of the whole interview for
+analysis later.
+
+You can turn on background audio recording in the Form Builder by clicking on
+Layout & Settings.
+
+![Backgrouns audio](images/photo_audio_video_file/background_audio.png)
+
+In XLSForm, you can add the background recording feature by using the
+background-audio meta question type as follows:
+
+| type             | name             | label |
+| :--------------- | :--------------- | :---- |
+| background-audio | background_audio |       |
+
+<section class="note">
+
+Note: It is not possible to record audio using “Audio” question type while
+background audio recording is underway on the form. When background audio
+recording is on, all “Audio” question types are deactivated.
+
+</section>
+
+Read more about background audio recording
+[on the linked article](recording-interviews.md).
+
+## Lowering the file size of collected media
+
+If your form will be collecting a lot of media, you might find it difficult to
+upload them to the KoboToolbox Server or download the media attachments from it.
+This is why it is always a good idea to manage the file sizes of collected media
+files such as images and videos.
+
+You can define the maximum size of images you collect using the “Photo” question
+type by going to the question settings and setting the “max-pixels” setting in
+the formbuilder.
+
+![Max pixels](images/photo_audio_video_file/max-pixels.png)
+
+In XLSForm, you can do the same by adding max-pixels in the parameters column as
+follows:
+
+| type  | name  | label         | parameters     |
+| :---- | :---- | :------------ | :------------- |
+| photo | photo | Capture photo | max-pixels=480 |
+
+In KoboCollect, you can also choose the video quality and photo size through the
+Form Management section of the project settings.
+
+Read more about how to lower file sizes in
+[this linked article](lower_file_size.md).
+
+## Limiting uploadable file types for “File” question type
+
+When you add a “File” question type, any type of file can be uploaded by the
+person entering the data. In order to control the kind of files that can be
```

Comment:
Rather something like this for the first sentence:
> All file types are accepted by default for the "File" question type.

Reply:
Corrected